app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/EditViewModel.java

```diff
@@ -12,15 +12,18 @@
 
     private final EditCompoundRecyclerView editCompoundRecyclerView;
     private final TagsManager tagsManager;
+    private final EditTagListener editTagListener;
 
     EditViewModel(String title,
                   String imageUrl,
                   EditCompoundRecyclerView editCompoundRecyclerView,
-                  TagsManager tagsManager) {
+                  TagsManager tagsManager,
+                  EditTagListener editTagListener) {
```

Comment:
editTagListener is redundant, listener is enough,
tagsManager -> manager
EditCompoundRecyclerView -> recyclerView (not part of this commit)